bwapi/bwresources.py

```diff
@@ -168,13 +211,14 @@ def delete_all(self, names):
         Args:
             names:   A list of the names of the queries that you'd like to delete.
         """
-        for name in names:
-            if name in self.ids:
-                resource_id = self.ids[name]
-                self.project.delete(
-                    endpoint=self.specific_endpoint + "/" + str(resource_id)
-                )
-                logger.info("{} {} deleted".format(self.resource_type, name))
+        pks = [self.get_resource_id(x) for x in names]
+
+        for pk in pks:
```

Comment:
NABD but `pk` and `pks` aren't terribly descriptive variable names

Reply:
Changed to resource_id